cylance_connector.py

```diff
@@ -292,17 +292,13 @@ def _make_rest_call(self, url, action_result, headers=None, params=None, json=No
         resp_json = None
 
         try:
-            request_func = getattr(requests, method)
-        except AttributeError:
-            return RetVal(action_result.set_status(phantom.APP_ERROR, "Invalid method: {0}".format(method)), resp_json)
-
-        try:
-            r = request_func(
-                            url,
-                            json=json,
-                            data=data,
-                            headers=headers,
-                            params=params)
+            kwargs = {
+                "json": json,
+                "data": data,
+                "headers": headers,
+                "params": params,
+            }
+            r = requests.request(method, url, **kwargs)
```

Comment:
did doing this resolve the NoneType error?

Reply:
Yes. https://cd.splunkdev.com/phantom-apps/app-tests/-/pipelines/6360886 
This approach was also suggested by Parv Desai from CREST per https://splunk.atlassian.net/browse/SOARHELP-1006?focusedCommentId=9704587

Reply:
interesting. I wonder if this is just bypassing the usage tracker call the platform team added entirely by not calling that function block in the platform code.